From f417e74618898e15e9e9c0f226670048dea517d7 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Tue, 17 Dec 2019 18:26:03 -0700
Subject: [PATCH 10/22] PCI: altera: Adjust indentation in
 altera_pcie_valid_device

Clang warns:

../drivers/pci/controller/pcie-altera.c:196:3: warning: misleading
indentation; statement is not part of the previous 'if'
[-Wmisleading-indentation]
         return true;
         ^
../drivers/pci/controller/pcie-altera.c:193:2: note: previous statement
is here
        if (bus->number == pcie->root_bus_nr && dev > 0)
        ^
1 warning generated.

This warning occurs because there is a space after the tab on this line.
Remove it so that the indentation is consistent with the Linux kernel
coding style and clang no longer warns.

Fixes: eaa6111b70a7 ("PCI: altera: Add Altera PCIe host controller driver")
Link: https://github.com/ClangBuiltLinux/linux/issues/815
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 drivers/pci/controller/pcie-altera.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pci/controller/pcie-altera.c b/drivers/pci/controller/pcie-altera.c
index b447c3e4abad..24cb1c331058 100644
--- a/drivers/pci/controller/pcie-altera.c
+++ b/drivers/pci/controller/pcie-altera.c
@@ -193,7 +193,7 @@ static bool altera_pcie_valid_device(struct altera_pcie *pcie,
 	if (bus->number == pcie->root_bus_nr && dev > 0)
 		return false;
 
-	 return true;
+	return true;
 }
 
 static int tlp_read_packet(struct altera_pcie *pcie, u32 *value)
-- 
2.24.1

